### ISSUE TITLE
NDK-build faild 

### ISSUE THREAD

**Issue by ritesh-bhavsar86** [uberspot/opentee-android#7]
i'm running otservice module then i got below error :

/Users/Anand/Documents/OpenTEE/android-ndk-r15b/build/core/build-binary.mk
Error:(688) Android NDK: Module nativelibtee-jni depends on undefined modules: c    
Error:(701) *** Android NDK: Aborting (set APP_ALLOW_MISSING_DEPS=true to allow missing dependencies)    .  Stop.
:otservice:buildNative FAILED

can you please suggest solution for this,

Also give steps for project setup and configuration

**Comment by lordqwerty** [comment]
Hi Ritesh,
I am not a developer on this project but there is documentation available https://open-tee.github.io/documentation/

Hope this helps!

**Comment by lordqwerty** [comment]
More specifically 
https://open-tee.github.io/android/

**Comment by ritesh-bhavsar86** [comment]
@lordqwerty  you are correct but this will not removing my error , Guys any one help me to sort out issues

**Comment by ritesh-bhavsar86** [comment]
/Users/Anand/Documents/OpenTEE/tee/opentee-android/otservice/src/main/jni/LibteeWrapper.cpp
Error:(17, 35) tee_shared_data_types.h: No such file or directory
compilation terminated.    please tell us where i did wrong
